plugins/generator-1.20.6/neoforge-1.20.6/templates/elementinits/potioneffects.java.ftl

```diff
@@ -65,7 +64,7 @@ package ${package}.init;
 
 	private static void expireEffects(Entity entity, MobEffectInstance effectInstance) {
 		<#compress>
-		MobEffect effect = effectInstance.getEffect();
+		MobEffect effect = effectInstance.getEffect().value();
 		<#list effects_that_expire as effect>
 		if (effect == ${effect.getModElement().getRegistryNameUpper()}.get()) {

```

Comment:
Since you now have holder from `effectInstance.getEffect()`, more correct solution would be to use "is"

Reply:
The code you now did is not something we would consider valid Java syntax:

`.is(Holder<MobEffect>)`

Reply:
I know, because I don't know how to make it work

Reply:
See 1.20.6 brewing recipe template for inspiration on how to use "is"

Reply:
Ok thank you